README.md

```diff
@@ -63,6 +63,14 @@ $ yarn bootstrap # Bootstrap with lerna
 $ yarn test # Run tests
 ```
 
+Commit:
+
+```bash
+$ git add .
+$ yarn commit
+$ npx git-cz # if you are using NPM 5.2+
```

Comment:
If you already added the commit script to allow ```yarn commit```, I don't think mentioning npx is necessary. Plus with yarn, you can simply do ```yarn git-cz``` and it will do the same thing as npx.